src/features/content-explorer/content-explorer/ContentExplorerBreadcrumbs.js

```diff
@@ -28,23 +29,25 @@ const ContentExplorerBreadcrumbs = ({
         >
             <IconChevron direction="left" size="6px" color="#333" />
         </Button>
-        <Breadcrumb label={formatMessage(messages.breadcrumb)}>
+        <Breadcrumb label={formatMessage(messages.breadcrumb)} {...breadcrumbProps}>
             {foldersPath.map((folder, i) => (
-                <PlainButton
-                    className="lnk"
-                    key={folder.id}
-                    title={folder.name}
-                    onClick={event => onBreadcrumbClick(i, event)}
-                >
-                    {i === 0 && <IconAllFiles />}
-                    <span>{folder.name}</span>
-                </PlainButton>
+                <div key={folder.id} className="lnk">
+                    <PlainButton
+                        className="breadcrumb-lnk"
```

Comment:
Do we need this class? Is it doing anything?

Reply:
I changed this to a test-id attr since it is only used within a testing context